app/views/welcome/index.html.erb

```diff
@@ -9,7 +8,6 @@
       <div class="thumbnail">
         <img src="http://upload.wikimedia.org/wikipedia/commons/4/4d/Cat_November_2010-1a.jpg">
       </div>
-      <h5 class="cat-name"><%= category.name %></h5>
```

Comment:
Can you explain your thinking for removing this? My recollection is that we talked about including the category name at the bottom of each project, kind of as a tag.